lib/AssemblyUtil/AssemblyToFasta.py

```diff
@@ -0,0 +1,103 @@
+import os
+import shutil
+
+from Bio import SeqIO
+from Bio.Seq import Seq
+from Bio.SeqRecord import SeqRecord
+from Bio.Alphabet import SingleLetterAlphabet
+
+from DataFileUtil.DataFileUtilClient import DataFileUtil
+
+
+class AssemblyToFasta:
+
+    def __init__(self, callback_url, scratch):
+        self.scratch = scratch
+        self.dfu = DataFileUtil(callback_url)
+
+
+    def export_as_fasta(self, ctx, params):
+        ''' Used almost exclusively for download only '''
+        # validate parameters
+        if 'input_ref' not in params:
+            raise ValueError('Cannot export Assembly- not input_ref field defined.')
+
+        # export to a file
+        file = self.assembly_as_fasta(ctx, {'ref': params['input_ref']})
+
+        # create the output directory and move the file there
+        export_package_dir = os.path.join(self.scratch, file['assembly_name'])
+        os.makedirs(export_package_dir)
+        shutil.move(file['path'], os.path.join(export_package_dir, os.path.basename(file['path'])))
```

Comment:
Just to let the 'assembly_as_fasta' come up with the file name.  I guess it could probably be refactored to save the file directly to the export package directory, but probably not worth the added complexity of trying to figure out a good name ahead of time.